msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -364,6 +364,27 @@ The default value of the property `isChatResourceSpecificConsentEnabled` is base
 <br>
```

Comment:
Any section that mentions "team or chat" should now also mention user. One example is "Install your app in a team or chat" section, which should say "Install your app in a team, chat, or user"

Reply:
Updated.